file-server/nginx/nginx-ssl.site.default.conf

```
@@ -0,0 +1,19 @@
+server {
+    listen       80;
+    listen      443 ssl http2;
+    listen [::]:443 ssl http2;
+    server_name  localhost;
+
+    ssl_certificate /etc/nginx/certs/snakeoil.cert.pem;
+    ssl_certificate_key /etc/nginx/certs/snakeoil.key.pem;
+    ssl_session_timeout 10m;
+    ssl_protocols TLSv1.2 TLSv1.1 TLSv1;
+    # See: https://github.com/dev-sec/nginx-baseline/blob/af883f35d86ed95a6f41ef7fdfdfc1b25a249273/controls/nginx_spec.rb#L227-L239
+    ssl_ciphers 'ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256';
+    ssl_prefer_server_ciphers on;
+
+    location / {
+        proxy_set_header X-Forwarded-Proto $scheme;
+        proxy_pass http://server:5000;
```

This is needed so that `server` can redirect properly (esp important for Twitter and Facebook OAuth, which otherwise get redirected so `//server:5000/home` instead of `//example.com/home`

```suggestion
        proxy_set_header X-Forwarded-Proto $scheme;
        proxy_set_header Host $host;
        proxy_pass http://server:5000;
```

---

Fixed!